fixtures/services/bat-safety.yaml

```diff
@@ -1,18 +1,21 @@
 slug: bat-safety
 
-topic: healthcare-and-prevention
-
+topic: healthcare-prevention
+image: curbside-compost
 contact: 311
 
 title_en: Keep yourself safe around bats in Austin
+title_es: Keep yourself safe around bats in Austin
```

Comment:
@LauraTrujillo  title is required for all the translations so I'm just using the english title as a placeholder